test.py

```diff
@@ -0,0 +1,4 @@
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import Gst
+print(Gst)
```

Comment:
Ack that file wasn't supposed to come along. Oh well but yes @matysek I can add it to the tests and I'll add tests for the other modules as well.